.github/workflows/pr-cmd-format.yml

```
@@ -0,0 +1,53 @@
+
+on:
+  issue_comment:
+    types: [created]
+
+name: PR auto-formatting command
+
+jobs:
+  format:
+    if: ${{ github.event.issue.pull_request && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER' || github.event.issue.user.id == github.event.comment.user.id) && startsWith(github.event.comment.body, '/format') }}
+    name: auto-format
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the repository
+        uses: actions/checkout@v2
```

v3 is the latest version now
```suggestion
        uses: actions/checkout@v3
```